tests/lax_autodiff_test.py

```diff
@@ -195,38 +195,43 @@ def check_grads_bilinear(f, args, order,
 
 class LaxAutodiffTest(jtu.JaxTestCase):
 
-  @parameterized.named_parameters(itertools.chain.from_iterable(
-      jtu.cases_from_list(
-        {"testcase_name": jtu.format_test_name_suffix(
-            rec.name, shapes, itertools.repeat(dtype)),
-         "op": rec.op, "rng_factory": rec.rng_factory, "shapes": shapes, "dtype": dtype,
-         "order": rec.order, "tol": rec.tol}
-        for shape_group in compatible_shapes
+  @parameterized.parameters(itertools.chain.from_iterable(
+    jtu.sample_product_testcases(
+      [dict(op=rec.op, rng_factory=rec.rng_factory, order=rec.order, tol=rec.tol)],
```

Comment:
could we just pass `rec=[rec]` here instead and pull out its contents in the test function? I think the reason we have split this in the past is because `rec` doesn't format well in the name, but now that we're not generating any name we could simplify the parametrization. What do you think?

Reply:
I'm going to punt this to a future PR. If it makes sense to do here, it probably makes sense to do in a number of other tests that have a similar pattern.